File: ex2/plotData.m
function plotData(X, y)
% Plots the data points X and y
%    X is assumed to be a Mx2 matrix.

    figure;
    hold on;
    pos = find( y == 1 );
    neg = find( y == 0 );

    plot( X(pos, 1), X(pos, 2), 'k+','LineWidth', 2, 'MarkerSize', 7);
    plot( X(neg, 1), X(neg, 2), 'ko', 'MarkerFaceColor', 'y','MarkerSize', 7);
    hold off;

end

File: Logistic Regression/python/plotData.py
import matplotlib.pyplot as plt
def plotData(X, y):
#  X is assumed to be a Mx2 matrix.
    plt.figure

    pos = X[ y == 1 ]
    neg = X[ y == 0 ]
    
    plt.plot(pos.iloc[:, 0], pos.iloc[:, 1], 'k+', linewidth=2, markersize=7)
    plt.plot(neg.iloc[:, 0], neg.iloc[:, 1], 'ko', markerfacecolor='y', markersize=7)
    return